### PR TITLE
Restructure README to include usage as a library

### PR DESCRIPTION
**Pull request by deathbird**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,60 @@
...
+Transifex Python Toolkit (referred as **the Toolkit** for brevity) is a collection of tools that allow you to easily localize your applications using Transifex (TX). The toolkit features fetching translations over the air (OTA) to your apps.
+
+Using Transifex (TX) the localization workflow typically consists of the following steps:
+1. Find - extract the source strings in your source code
+2. Push them into a Transifex project
+3. In Transifex your source strings get translated by translators in target languages
+4. Retrieve a string translation of a target language from Transifex to display it
+
+Transifex Python Toolkit is used in steps 1, 2, 4 above. Possible usage scenarios:
+* [In a Django web application](#scenario-1-integrate-with-a-django-app) The toolkit is fully integrated in Django framework. It contains utilities to mark your source strings for translation and display translation in templates or views. It also provides special CLI commands for pushing to TX and even migrating your existing template code to the new syntax.
```
Comment (sergafts) [inline README.md:15]: Let's add the bold
...to mark your source strings for translation, **retrieve the translated strings** and display translation**s** in templates or views.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,60 @@
...
 ```python
 pip install transifex-python
 ```
 
-The toolkit provides a Python interface for preparing your code for localization, as well as a thin Django wrapper for Django apps.
+# Transifex Setup
+Before you begin using the Toolkit, you will also need an account in [Transifex](https://www.transifex.com) and a project.
+To set a project compatible with this toolkit contact [support](https://www.transifex.com/contact/) and you
+will be given a set of credentials (a public token and a secret), that you can use in your code for authentication.
+We will refer to these credentials in the text below as:
```
Comment (sergafts) [inline README.md:37]: Since those are only referenced in the plain python section, I'd remove this note here.

Comment (deathbird) [reply README.md:37]: They are also referenced in the Setup Django section. That's why I extracted them in a common  setup section. I also made minor amendments to in the Python lib examples to reference more explicitly the values of these credentials.

Comment (sergafts) [reply README.md:37]: Ah. Right. Okay then :)

**Pull request merged**